src/software/ai/hl/stp/play/ball_placement/ball_placement_play_fsm.cpp

```diff
@@ -273,14 +273,14 @@ Angle BallPlacementPlayFSM::calculateWallKickoffAngle(const Point &ball_pos,
 
 void BallPlacementPlayFSM::setupMoveTactics(const Update &event)
 {
-    // assign all but one of the robots to line up away from the ball placing robot
-    unsigned int num_move_tactics = event.common.num_tactics - 1;
-
-    if (num_move_tactics == 0)
+    if (event.common.num_tactics <= 1)
     {
         return;
     }
 
+    // assign all but one of the robots to line up away from the ball placing robot
+    unsigned int num_move_tactics = event.common.num_tactics - 1;
+
```

Comment:
thoughts on adding a simulated test for this situation for regression purposes

making a new issue here to address this is fine with me if you don't want to do it right now

Reply:
also consider replacing this with an `int` or we can make a new issue that replaces all `unsigned` uses with `int`

Reply:
I've updated it to use `int` and added #3228 for addressing across other plays as well